### PR TITLE
Fix [Batch run] Display selected function on return to step 1

### PR DESCRIPTION
**Pull request by pinis-gini-apps**
- Batch run: Display selected function on return to step 1
- Gira: https://jira.iguazeng.com/browse/ML-4742
- Related To: https://github.com/pinis-gini-apps/ui/pull/1

### CODE REVIEW

File: src/lib/components/Wizard/Wizard.scss [hunk 1]
```diff
@@ -28,5 +28,9 @@
     .wizard-form__hidden-content-item {
       display: none;
     }
+    
+    .wizard-form__visible-content-item{
+      height: 100%;
+    }
```
Comment (mavdryk) [inline src/lib/components/Wizard/Wizard.scss]: use formatting before commit. There should be a space before `{`

**Review by mavdryk — CHANGES_REQUESTED** [review]

File: src/lib/components/Wizard/Wizard.scss (hunk 1 above)
Comment (pinis-gini-apps) [reply src/lib/components/Wizard/Wizard.scss]: @mavdryk done.

**Comment by pinis-gini-apps** [comment]
@mavdryk ready for review.

File: src/lib/components/Wizard/Wizard.scss [hunk 2]
```diff
@@ -26,7 +26,14 @@
     height: 100%;
 
     .wizard-form__hidden-content-item {
-      display: none;
+      position: absolute;
+      visibility: hidden;
+      opacity: 0;
```
Comment (mavdryk) [inline src/lib/components/Wizard/Wizard.scss:32]: Let's add also `height: 0` here

**Review by mavdryk — CHANGES_REQUESTED** [review]
.

File: src/lib/components/Wizard/Wizard.scss (hunk 2 above)
Comment (pinis-gini-apps) [reply src/lib/components/Wizard/Wizard.scss:32]: done.

**Pull request closed without merging**